File: Python/Calculadora_V1.py
print('Calculadora')
a = int(input('Ingrese un numero: '))
print('[1] Multiplicacion [2] Sumar [3]Resta [4] Division')
operador = int(input('Ingrese alguno de los siguientes operadores con base al numero: '))
b = int(input('Ingrese otro numero: '))
if operador == 1:
    operador = (a*b)
    print('el resultado de la multiplicacion', a,'*',b, 'es:', operador)
elif operador == 2:
    operador = (a+b)
    print('el resultado de la suma', a,'+',b, 'es:', operador)
elif operador == 3:
    operador = (a-b)
    print('el resultado de la resta', a,'-',b, 'es:', operador)
elif operador == 4:
    operador = (a/b)
    print('el resultado de la division', a,'/',b, 'es:', operador)
else:
    print('Usted ingreso un operador diferente')
print('Fin de la calculadora')

File: Js/Calculadora_V1.js
alert('Calculadora Experta');
let a = parseInt(prompt('Ingrese un numero'));
let operador = prompt('Ingrese un operador: [1]Multiplicacion [2]suma [3]Resta [4]Division');                        
let b = parseInt(prompt('ingrese otro numero'));
if (operador == '1'){
    resultado = (a*b)
    alert(`El resultado de la multiplicacion ${a} x ${b} es: ${resultado}`)
} else if (operador == '2') {
    resultado = (a+b)
    alert(`El resultado de la suma ${a} + ${b} es: ${resultado}`)
} else if (operador == '3') {
    resultado = (a-b)
    alert(`El resultado de la resta ${a} - ${b} es: ${resultado}`)
} else if (operador == '4') {
    resultado = (a/b)
    alert(`El resultado de la division de ${a} / ${b} es: ${resultado}`)
} else {
    alert(`Usted escogio un operador diferente al mostrado`)
}
    
